src/main/scala/org/camunda/feel/impl/parser/FeelParser.scala

```diff
@@ -18,21 +18,34 @@ package org.camunda.feel.impl.parser
 
 import fastparse.JavaWhitespace._
 import fastparse._
+import fastparse.internal.Logger
```

Comment:
Considering that all uses of the logger have been removed, this may also be removed again